product_docs/docs/pgd/5/node_management/logical_standby_nodes.mdx

```diff
@@ -11,54 +11,46 @@ A master node can have zero, one, or more logical standby nodes.
     between data centers is a concern; otherwise having more data nodes per
     location is always preferred.
 
-With a physical standby node, the node never comes up fully, forcing it to
-stay in continual recovery mode.
-PGD allows something similar. [`bdr.join_node_group`](/pgd/latest/reference/nodes-management-interfaces#bdrjoin_node_group) has the `pause_in_standby`
+Logical standby nodes are nodes which are held in a state of continual recovery,
+constantly updating until they are required. [`bdr.join_node_group`](/pgd/latest/reference/nodes-management-interfaces#bdrjoin_node_group) has the `pause_in_standby`
```

Comment:
I'd expect at least some customers knowing Postgres, so the comparison with (Postgres') physical standby node seems worthwhile to me. Won't those people otherwise keep wondering how it compares?

Reply:
```suggestion
Logical standby nodes are nodes which are held in a state of continual recovery,
constantly updating until they are required. This is similar to Postgres physical standbys operate, but uses logical replication for better performance. [`bdr.join_node_group`](/pgd/latest/reference/nodes-management-interfaces#bdrjoin_node_group) has the `pause_in_standby`
```

Reply:
LGTM